xml/windows-ses.xml

```diff
@@ -223,6 +224,26 @@ C:\ProgramData\ceph\ceph.client.admin.keyring
       Add-VMHardDiskDrive -VMName BootFromRBD -DiskNumber $diskNumber
       Start-VM -VMName BootFromRBD
 </screen>
+  <note>
+   <para>
+    To avoid all disks going online automatically, the following setting
```

Comment:
```suggestion
    To avoid all disks by going online automatically, the following setting

Reply:
It probably makes more sense to say "To prevent all disks from...". I'll make that change manually.